marklogic-data-hub-central/ui/e2e/cypress/integration/curation/curate/mapping.spec.tsx

```diff
@@ -41,6 +41,7 @@ describe('Mapping', () => {
     toolbar.getLoadToolbarIcon().click();
     cy.waitUntil(() => loadPage.stepName('ingestion-step').should('be.visible'));
     loadPage.addNewButton('card').click();
+    loadPage.saveButton.should('be.enabled');
```

Comment:
Looks like there is a cypress failure on this line. Please modify the code to be 
`loadPage.saveButton().should('be.enabled');`
That should fix it.

Reply:
thanks